requirements.txt

```diff
@@ -19,3 +19,4 @@ recommonmark==0.4.0
 six==1.10.0               # via sphinx
 snowballstemmer==1.2.1    # via sphinx
 Sphinx==1.4.1
+sphinx-reload==0.2.0
```

Comment:
Also can you make a requirements-dev.txt file. This probably shouldn't live in the requirements.txt file